jax/_src/random.py

```diff
@@ -2216,3 +2216,110 @@ def _geometric(key, p, shape, dtype) -> Array:
   log_one_minus_p = jnp.broadcast_to(log_one_minus_p, shape)
   g = lax.floor(lax.div(log_u, log_one_minus_p)) + 1
   return g.astype(dtype)
+
+
+def triangular(key: KeyArray,
+               left: RealArray,
+               mode: RealArray,
+               right: RealArray,
+               shape: Optional[Shape] = None,
+               dtype: DTypeLikeFloat = dtypes.float_) -> Array:
+  r"""Sample Triangular random values with given shape and float dtype.
+
+  The values are returned according to the probability density function:
+
+  .. math::
+      f(x; a, b, c) = \frac{2}{c-a} \left\{ \begin{array}{ll} \frac{x-a}{b-a} & a \leq x \leq b \\ \frac{c-x}{c-b} & b \leq x \leq c \end{array} \right.
+
+  on the domain :math:`a \leq x \leq c`.
```

Comment:
I think a newline is required here before `Args:` – that's causing a formatting error in the readthedocs build.